### DISCUSSION TITLE
Modify Inactivity-leave for Party Plugin

### DISCUSSION THREAD

**Discussion by 1Defence** [runelite/runelite#16086] [Ideas]
Currently after 30 minutes of being [logged out,](https://github.com/runelite/runelite/blob/2835bef29fd67604e6154d7dc0185b1e34fb66c2/runelite-client/src/main/java/net/runelite/client/plugins/party/PartyPlugin.java#L368) your party session is terminated for being considered inactive. 

What I'm suggesting is on login, party checks if you were previously inactive-kicked and automatically rejoins the previous party if that condition is met.

It wouldn't occur on initial client start this way, but you wouldn't have to rejoin on numerous clients every time you take a break with this change.

**Comment by Nightfirecat** [comment]
We're not really interested in doing any kind of automatic (re-)joining of parties as there are server-side resource limitations for us to provide the party system. The party panel has a "Join previous party" which we recommend using for this.

**Reply by 1Defence** [reply]
Initially thought this would suffice due to not occurring on initial start but that's understandable.

Perhaps another solution could be adding a hotkey to manually perform the same functionality of the "join previous party" button?

I've been using that button already it just becomes problematic when you're on a lot of clients and have no indication that your party has ended and have to scramble to open side panels of 8+ accounts mid-combat.
